source/common/tracing/zipkin/zipkin_tracer_impl.cc

```diff
@@ -81,30 +79,17 @@ Tracing::SpanPtr Driver::startSpan(const Tracing::Config& config, Http::HeaderMa
                                    const std::string&, SystemTime start_time) {
   Tracer& tracer = *tls_->getTyped<TlsTracer>().tracer_;
   SpanPtr new_zipkin_span;
+  bool sampled(true);
```

Comment:
https://www.envoyproxy.io/docs/envoy/latest/configuration/http_conn_man/headers.html#x-b3-sampled says 
"The x-b3-sampled HTTP header is used by the Zipkin tracer in Envoy. When the Sampled flag is 1, the span will be reported to the tracing system"

From this I would infer if the header was absent since the sampled flag isn't 1, it would not be reported.
I think sampling by default would be more consistent with the prior experience but do you think it's worth a docs upate "Unless the Sampled flag is 0, the span will be reported"?

Reply:
Will try to make this clear in the doc update.